### PR TITLE
feat: pulse current bid when it updates via polling

### PR DESCRIPTION
**Pull request by mzikherman**
Looks like:

polling-pulse

I'll make a couple of comments inline, but this wound up being relatively straightforward (I had thought there might be more of a refactor needed).

### CODE REVIEW

File: src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx [hunk 1]
```diff
@@ -28,11 +45,15 @@ export const ArtworkSidebarAuctionPolling: React.FC<Props> = ({
     },
     intervalTime: 10000,
     key: artwork.internalID,
+    clearWhen: isClosed,
```
Comment (mzikherman) [inline src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx:48]: This was an unrelated update (probably should have been its own commit, but was so minor). 

The `usePoll` hook has a `clearWhen` property which will not perform polling (or stop if it was) when true (in https://github.com/artsy/force/blob/df3c95874980cc6806afa867207790cb1548fe2e/src/v2/Apps/Conversation/Utils/usePoll.ts#L30)

I realized we were making these polling requests when on a lot page where the auction was already closed, which is a lot of extra requests that aren't needed.

File: src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx [hunk 2]
```diff
@@ -17,6 +18,22 @@ export const ArtworkSidebarAuctionPolling: React.FC<Props> = ({
...
+  const isClosed = !!sale?.isClosed
+  const currentBidDisplay = saleArtwork?.currentBid?.display
+
+  const [currentBidChanged, setCurrentBidChanged] = useState(false)
+
+  const isMounted = useRef(false)
+
+  useEffect(() => {
+    if (isMounted.current) {
+      setCurrentBidChanged(true)
```
Comment (mzikherman) [inline src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx:31]: Essentially, when the `currentBidDisplay` changes, _and_ the component has already mounted (so skipping the initial rendering), set this to true.

File: src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx [hunk 3]
```diff
@@ -129,9 +152,10 @@ export const ArtworkSidebarCurrentBidInfo: React.FC<ArtworkSidebarCurrentBidInfo
             </Box>
           )}
 
-          <Text variant="lg" pl={0.5}>
+          {/* force a different key on each re-render so animation triggers */}
+          <CurrentBid key={Math.random()} variant={"lg"} pl={0.5}>
```
Comment (mzikherman) [inline src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx:156]: A new key forces a re-render of this element, which is to enable playing the animation once, _each time there is a re-render_. Without it, the animation plays once the first time this component re-renders with an updated current bid, but then subsequent times as the bid is increasing, it doesn't play.

**Review by nspinazz89 — APPROVED** [review]
Looks good to me!

**Pull request merged**